packages/appkit/src/universal-adapter/client.ts

```diff
@@ -464,6 +469,8 @@ export class UniversalAdapterClient {
       SafeLocalStorage.removeItem(SafeLocalStorageKeys.WALLET_ID)
       SafeLocalStorage.removeItem(SafeLocalStorageKeys.ACTIVE_CAIP_NETWORK)
 
+      AccountController.setIsConnected(false, this.appKit?.getActiveChainNamespace() || 'eip155')
```

Comment:
These `setIsConnected` calls should be already removed completely, is this branch up to date?

Reply:
Mhmm yeah the branch wasn't up to date